Dockerfile

```diff
@@ -1,27 +1,25 @@
+#building bot
 FROM gcc:latest as build
 
 RUN apt-get update && \
     apt-get install -y \
     libboost-dev libboost-system-dev \
     cmake 
 
+
 RUN apt-get install g++ make binutils libboost-system-dev libssl-dev zlib1g-dev libcurl4-openssl-dev
 
+#installing tgbot lib
```

Comment:
Maybe add a space between comment token and text to improve readability.

Reply:
fixed